### PR TITLE
Fix implicit declaration of function strcasecmp with musl

### PR DESCRIPTION
**Pull request by listout**

### CODE REVIEW

**Comment by derselbst** [comment]
Well yeah, that's the same fix as mine in 0b40da5 . I would be more interested in a way to reproduce this build failure in a CI pipeline, particularly in #1243. But the Alpine build succeeds there: https://dev.azure.com/tommbrt/tommbrt/_build/results?buildId=9127

I don't see how the question of C99 compliance is relevant here, esp. because fluidsynth is a C89 project currently.

**Comment by listout** [comment]
> I don't see how the question of C99 compliance is relevant here, esp. because fluidsynth is a C89 project currently.

You're correct, by bad. Let me correct that.

**Comment by derselbst** [comment]
Ok, thanks for the pointer. I'll double check any compiler flags with the apline build when I find the time.

According to #1229 this is a problem on Alpine as well, but this issue lacks precise instructions on reproducing it.

File: CMakeLists.txt [hunk 1]
```diff
@@ -159,6 +159,7 @@ include ( CheckIncludeFile )
 include ( CheckSymbolExists )
 include ( CheckTypeSize )
 check_include_file ( string.h HAVE_STRING_H )
+check_include_file ( string.h HAVE_STRINGS_H )
```
Comment (sevz17) [inline CMakeLists.txt:162]: This should be `strings.h`

Comment (thesamesam) [reply CMakeLists.txt:162]: whoops, thanks

**Comment by derselbst** [comment]
Superseded by #1245

**Pull request closed without merging**